README.md

```diff
@@ -0,0 +1,82 @@
+# Configurable LTI Consumer XBlock
+
+This XBlock is used to replace edX's lti_consumer xblock. It makes available to
+conceptors in CMS several preconfigured versions of LTI Consumer original
+XBlock. Preconfigured fields are no longer modifiable.
+
+## Installation
+
+This package can be installed with `pip`:
+
+```bash
+$ pip install [--process-dependency-links] configurable_lti_consumer-xblock
```

Comment:
the "-" in the name, is it how it's done in open edx xblock?

Reply:
Yes... :speak_no_evil: